audio/softsynth/fluidsynth.cpp

```diff
@@ -179,7 +184,16 @@ int MidiDriver_FluidSynth::open() {
 
 	const char *soundfont = ConfMan.get("soundfont").c_str();
 
+#ifdef IPHONE_OFFICIAL
```

Comment:
This whole block needs a `FIXME/HACK` comment. It indicates that the chroot FS implementation breaks functionality in unexpected ways. In the future we need to get rid of this special code for iOS.